morf-core/src/main/java/org/alfasoftware/morf/metadata/SchemaUtils.java

```diff
@@ -553,6 +573,15 @@ public TableBuilder columns(Column... columns) {
     }
 
 
+    /**
+     * @see org.alfasoftware.morf.metadata.SchemaUtils.TableBuilder#columns(java.lang.Iterable)
+     */
+    @Override
+    public TableBuilder columns(Iterable<? extends Column> columns) {
+      return new TableBuilderImpl(getName(), columns, indexes(), isTemporary());

```

Comment:
Can we make columns(Column... columns) call this method since we construct a list in it.

Reply:
Makes sense.. But then the constructor's code might become untested. I'll try..

Reply:
Correction: There is no array-based constructor.